heralding/capabilities/ssh.py

```diff
@@ -65,6 +70,13 @@ def handle_connection(self):
         else:
             self.session = self.create_session(self.address, self.dest_address)
 
+    def get_auxiliary_info(self):
+        conn = self.connections_list[-1]
+        data_fields = AuxiliaryData.get_data_fields('ssh')
+        data = {i: conn.get_extra_info(i) for i in data_fields}
+        logger.debug("Auxiliary info: %s" % str(data))
```

Comment:
I think, this should go to `add_auxiliary_info` method. We will probably keep using `get_auxiliary_info` (with other implementations) in the other protocol classes and it would be better not to repeat `logger.debug("Auxiliary info: %s" % str(data))` in each implementation.

Reply:
No big deal!, i thought for debugging purpose logging the output to console may help. I will remove it.